src/components/fields/edit/InputRow.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Skeleton from '@material-ui/lab/Skeleton';
 
-import useLabel from '../../../hooks/useLabel';
+import useViewLabel from '../../../hooks/useViewLabel';
```

Comment:
gah I know this is a maze, but `<InputRow>` is actually usually (or maybe even always) paired with edit components, not view components. So the behavior here is fine but actually the language is a little wonky.